spec/features/amusement_parks/show_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "rails_helper"
+
+RSpec.describe "Amusment Park show page" do
+  before(:each) do
+    @park1 = AmusementPark.create!(name: "Denver", admission_cost: 33)
+    @ride1 = @park1.rides.create!(name: "Tower", thrill_rating: 8, open: true)
+    @ride2 = @park1.rides.create!(name: "Doom", thrill_rating: 8, open: false)
+    @ride3 = @park1.rides.create!(name: "Merry-go-round", thrill_rating: 3, open: true)
+    @mechanic1 = @ride1.mechanics.create!(name: "Martin", years_experience: 10)
+    @mechanic2 = @ride1.mechanics.create!(name: "Rodrigo", years_experience: 9)
+  end
+
+  describe '#US 3' do
+    it 'Shows all the mechanics that work in the amusment park' do
+      # When I visit an amusement park’s show page,
+      visit "/amusement_parks/#{@park1.id}"
+      # Then I see the name and price of admissions for that amusement park
+      expect(page).to have_content("Denver")
+      expect(page).to have_content(33)
```

Comment:
I feel like I've made this comment several times about these types of tests. When we look at line 19, what is our assertion? `expect(page).to have_content(33)` right? Now just looking at that assertion, does it make a whole lot of sense to just be looking for `33` on the page? Probably not. This assertion has the potential to create false positives and would not be considered a good test. Sure with the limited HTML in this IC we don't have any other numbers on the page, but what if we did, we're not writing our tests for that inevitable eventuality.

Here are two better assertions for numbers:

```ruby
within "#amusement_park_#{@@park1.id}" do
  expect(page).to have_content(33)
end
```

or 

```ruby
expect(page).to have_content("Admission Cost: $33")
```

Regardless of your choice, it would be best to make your assertions clear so that they don't mistakenly result in a false positive, especially when it comes to number values.